### DISCUSSION TITLE
SVG to RepliCAD JS code generator

### DISCUSSION THREAD

**Discussion by lf94** [sgenoud/replicad#32] [Show and tell]
Converted from my svg2curv code, enjoy https://github.com/lf94/svg2replicad

**Comment by sgenoud** [comment]
Thanks! I plan to include a `drawFromSVG` function in the library itself!

**Reply by phovorka** [reply]
Hi, i dont see it in library. Is there any other option how to extrude profile from SVG or DXF?

**Reply by raydeleu** [reply]
You can have a look at https://github.com/raydeleu/ReplicadManual/wiki/10.-Export-and-import as a work-around.

**Reply by alzatin** [reply]
> Thanks! I plan to include a `drawFromSVG` function in the library itself!

Hi! Are you still planning on adding drawFromSVG to the library?
